documentation/components/list/list-doc.html

```diff
@@ -0,0 +1,225 @@
+                        <div class="demo__list">
+                            <ul class="list">
+                                <li class="list__input m-xs-all">
+                                    <label for="id-1" class="center-content">
+                                        <input id="id-1" type="radio" class="m-xs-r" name="I1" checked>
+                                        Item-1
+                                    </label>
+                                </li>
+                                <li class="list__input m-xs-all">
+                                    <label for="id-2" class="center-content">
+                                        <input id="id-2" type="radio" class="m-xs-r" name="I1">
+                                        Item-2
+                                    </label>
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div class="code-ss">
+                        <iframe
+                            src="https://carbon.now.sh/embed?bg=rgba%28245%2C245%2C245%2C1%29&t=blackboard&wt=sharp&l=auto&width=966.5&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=16px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=15px&lh=124%25&si=false&es=2x&wm=false&code=%2520%253Cul%2520class%253D%2522list%2522%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522list__input%2520m-xs-all%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522id-1%2522%2520class%253D%2522center-content%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520id%253D%2522id-1%2522%2520type%253D%2522radio%2522%2520class%253D%2522m-xs-r%2522%2520name%253D%2522I1%2522%2520checked%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520Item-1%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Flabel%253E%250A%2520%2520%2520%2520%253C%252Fli%253E%250A%2520%2520%2520%2520%253Cli%2520class%253D%2522list__input%2520m-xs-all%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Clabel%2520for%253D%2522id-2%2522%2520class%253D%2522center-content%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%253Cinput%2520id%253D%2522id-2%2522%2520type%253D%2522radio%2522%2520class%253D%2522m-xs-r%2522%2520name%253D%2522I1%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520Item-2%250A%2520%2520%2520%2520%2520%2520%2520%2520%253C%252Flabel%253E%250A%2520%2520%2520%2520%253C%252Fli%253E%250A%253C%252Ful%253E"
+                            style="width: 100%; height: 366px; border:0; transform: scale(1); overflow:hidden;"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+                </div>
+            </div>
+        </main>
+    </div>
+    <script src="/documentation/docs.js"></script>
+</body>
+
+</html>
```

Comment:
You could add a blank line at the end to avoid these kinds of warnings from Github

Reply:
Due to auto file formatting!